src/components/Button/Button.module.css

```diff
@@ -21,6 +21,7 @@
   align-items: center;
   justify-content: center;
   gap: var(--spacing-gap-xs, 4px);
+  box-shadow: none;
```

Comment:
can you submit this change with a different PR?

the release note is generated with the PR titles and this is a relevant change that I'd not slip in the "footer standard props" pr :)

Reply:
oh, yes